sympy/core/relational.py

```diff
@@ -1075,9 +1079,218 @@ def _n2(a, b):
+                    return True
+                if is_ge(e, S.Zero):
+                    return is_ge(b, B)
+                if is_ge(S.Zero, e):
+                    return is_ge(B, b)
+            if is_ge(S.Zero, b):
+                m = Mod(e, 2)
+                if m == 0:
+                    if is_ge(e, S.Zero):
+                        return is_ge(b, B)
+                    if is_ge(S.Zero, e):
+                        return is_ge(B, b)
+                if m == 1:
+                    if is_ge(e, S.Zero):
+                        return is_ge(B, b)
+                    if is_ge(S.Zero, e):
+                        return is_ge(b, B)
+        # same base, b**e >= b**E
+        if b == B:
+            if b.is_zero:
+                return e.is_zero  # 1 >= {0, 1}
+            if b is S.NegativeOne:
+                return e.is_even  # 1 >= {-1, 1}
+            if b is S.One:
+                return True
+            if is_ge(S.NegativeOne, b):
+                m = Mod(e, 2)
+                M = Mod(E, 2)
+                if m in (0, 1) and M in (0, 1):
+                    if m != M:
+                        return m == 0  # pos > neg
+                    if is_ge(S.NegativeOne, b):
+                        if m == 1:
+                            return is_ge(E, e)
+                        return is_ge(e, E)  # m == 1
+                    if _small_neg(b):
+                        if m == 1:
+                            return is_ge(e, E)
+                        return is_ge(E, e)  # m == 1
+            elif _small_pos(b):
+                for v in (S.NegativeOne, S.Zero, S.One):
+                    if is_ge(v, e) and is_ge(E, v):
+                        return True
+            elif is_ge(b, S.One):
+                for v in (S.NegativeOne, S.Zero, S.One):
+                    if is_ge(e, v) and is_ge(v, E):
+                        return True
+
+    # make canonical since we didn't have a Pow for either side
+    ix = False  # keep track of whether L and R were exchanged
+    L, R = lhs, rhs
+    # move Rationals to the right
+    if L.is_Rational:
+        L, R = R, L
+        ix = ~ix
+    m, l, b = _aslinear(L)
+    mr, r, br = _aslinear(R)
+    if l.is_Pow or r.is_Pow:
+        L = l
+        R = mr/m*r + (br - b)/m
+        if m < 0:
+            ix = ~ix
+    if R.is_Number:
+        n, d = fraction(L)
+        if d is not S.One:
+            # bring denom to numerator on other side
+            # n/d ~ R -> n/R ~ d, watching for signs of R and d
+            if R != 0:
+                if R < 0:
+                    ix = ~ix
+                den = d
+                r = R
+            else:
+                den = S.Zero
+                r = S.One
+            if is_ge(d, S.Zero):
+                L, R = n/r, den
+            elif is_le(d, S.Zero):
+                ix = ~ix
+                L, R = n/r, den
+    # if we did more than change sign or switch sides, re-evaluate
+    if {L, R} - {lhs, rhs, -lhs, -rhs}:
```

Comment:
it seems to me that the return should be something like `is_ge(L,R)` if either is a power else `L >= R` if L and R are not trivially the same as at the start.